src/server/graphql.js

```diff
@@ -47,12 +43,10 @@ const resolverMap = {
     parseLiteral(ast) {
       if (ast.kind === Kind.INT) {
         return parseInt(ast.value, 10);
-      } else {
```

Comment:
I'm fine with it how it is, it just struck me as odd haha. Although, about 10 minutes after I wrote that comment, I wrote something almost exactly the same in Python, so it might be one of those things that's weird if I think about it too much but not otherwise.